src/grun-maciej/src/Fsd.Maciej.Cs/Fsd.Maciej.Cs.Ex2/Animals/Cat.cs

```diff
@@ -2,15 +2,9 @@
 {
     public class Cat : Animal
     {
-        public override string MakeSomeNois()
+        public Cat()
         {
-            return "Miauuuuuuu";
-        }
-
-        public override string AbautMe()
-        {
-            string _whoAmI = $"{base.AbautMe()}{Name}, {MakeSomeNois()}";
-            return _whoAmI;
+            base.MakeSomeNoise = "Miauuuuuuu";
```

Comment:
1. nie używamy raczej this i base gdy odwołujemy się do zmiennych
2. czy nie lepiej użyć odwołania do konstruktora klasy bazowej i przekazać ten parametr? public Cat() : base("Miauu") { }